composefiles/swarm-uniflow.yml

```diff
@@ -38,7 +38,7 @@ services:
       - PROXY_ENABLED=${PROXY_ENABLED:-false}
       - HTTP_PROXY=${HTTP_PROXY}
       - HTTPS_PROXY=${HTTPS_PROXY}
-      - NO_PROXY=workflow-proxy,frinx-frontend,inventory,${UNICONFIG_ZONES_LIST},unistore,uniresource
+      - NO_PROXY=workflow-proxy,frinx-frontend,inventory,krakend,unistore,uniresource,${UNICONFIG_ZONES_LIST},${NO_PROXY}
```

Comment:
Is ${NO_PROXY} ok? Isn't it recursion?

Reply:
It's ok. This configures NO_PROXY setting from host + additional settings to docker image.